spiceaidocs/docs/cli/reference/add.md

```diff
@@ -5,26 +5,41 @@ pagination_prev: null
 pagination_next: null
 ---
 
-Add Spicepod - adds a Spicepod to the project
+Adds a Spicepod to the project from a given package.
```

Comment:
We shouldn't introduce new terminology "package". In this case, keep it as `Spicepod`.